data_fusion/plugins/yaml/products/layered.yaml

```diff
@@ -42,13 +42,21 @@ spec:
             colorbar: True
     - name: Layered-Winds-Horizontal-Adjust
       source_names: ["layered"]
+      docstring: |
+        Layered winds product using horizontal adjust colorbar placement.
       product_defaults: Layered-Winds-Horizontal-Adjust
     - name: Layered-Winds-Vertical-Default
       source_names: ["layered"]
+      docstring: |
+        Layered winds product using Vertical Default colorbar placement.
       product_defaults: Layered-Winds-Vertical-Default
     - name: Layered-Winds-Vertical-Adjust
       source_names: ["layered"]
+      docstring: |
+        Layered winds product using Vertical Adjust colorbar placement.
       product_defaults: Layered-Winds-Vertical-Adjust
     - name: Layered-Winds-Horizontal-and-Vertical
       source_names: ["layered"]
+      docstring: |
+        Layered winds product using Horizontal and Vertical colorbar placement.
```

Comment:
These should have inherited the docstring from the product_defaults, correct?  It should not have to be defined here?

Reply:
With the architecture of create_plugin_registries, it is required. We grab information from each sub-product plugin, and if docstring doest't exist, we'll get a key error and this plugin won't be included. Even if they're minimal, I think it's important that we include them.

Reply:
Can you add logic to pull the docstring from product_defaults if it is not defined in the product during the plugin registry creation step ?

Reply:
This was fixed in geoips PR, so this PR will be closed as non planned